src/Umbraco.Web.UI.Client/src/views/stylesheets/edit.controller.js

```diff
@@ -171,78 +164,99 @@
 
             vm.stylesheet = stylesheet;
 
-            vm.setDirty = function () {
-                setFormState("dirty");
+            // Options to pass to code editor (VS-Code)
+            vm.codeEditorOptions = {
+                language: "css"
             }
 
-            //sync state
-            editorState.set(vm.stylesheet);
+            //monaco.languages.registerCompletionItemProvider().dispose();
```

Comment:
Do we still need this? :-)

Reply:
Good spot - it was my scratchpad for seeing what APIs are available in Monaco with their TypeScript definition file